test/Mono.Linker.Tests.Cases/BCLFeatures/ETW/CustomEventSource.cs

```diff
@@ -21,24 +22,30 @@ public static void Main ()
 	[EventSource (Name = "MyCompany")]
 	class MyCompanyEventSource : EventSource
 	{
+		[KeptMember (".ctor()")]
 		[Kept]
 		public class Keywords
 		{
 			[Kept]
 			public const EventKeywords Page = (EventKeywords) 1;
 
+			[Kept]
```

Comment:
I think we should link to https://github.com/mono/linker/issues/1174 - since once that is fixed, the annotation in EventSource can become just `PublicMethods | PublicNestedTypes` I think (or something similar, but much smaller than `All`).